src/amo/css/OverallRating.scss

```diff
@@ -1,29 +1,33 @@
+
 .OverallRating-choices {
   display: flex;
   justify-content: center;
 }
 
+$star-size: 46px;
+
 .OverallRating-choice {
-  font-size: 0.8em;
   font-weight: bold;
   cursor: pointer;
-  text-align: center;
-  padding-top: 90px;
-  width: 100px;
-  height: 100px;
-  background-size: 85px;
+  padding-top: 80px;
```

Comment:
This is just OCD but I do love me some alphabetised attributes, makes things so much easier to scan.